src/utils/url.ts

```diff
@@ -19,7 +19,7 @@ export const venueEntranceUrl = (venueId: string, step?: number) => {
 
 export const isExternalUrl = (url: string) => {
   try {
-    return new URL(url).host !== window.location.host;
+    return new URL(url, window.location.origin).host !== window.location.host;
```

Comment:
Could you please describe what is the difference here?
Why do we need to add it?

Reply:
We use this approach in other method, discussed here: https://github.com/sparkletown/sparkle/pull/1296#discussion_r604609316